### ISSUE TITLE
Mousing Anomalies On ChromeOS

### ISSUE THREAD

**Issue by MagFlux** [moonlight-stream/moonlight-chrome#324]
Hello,

On my Chromebook Pixel using the 0.7.4 extension I get weird and unusable mousing behavior.

In a gist, what happens is it appears there is some invisible smaller rectangular mousing region around a quarter of the size of my display that limits the range of the mouse pointer. Furthermore if you move the mouse pointer to one of these invisible edges it resets the mouse pointer to the center of the screen. This happens along all edges.

Everything else works perfectly and as expected as far as I can tell.

Oddly enough using the same extension on my Ubuntu Crouton install does not cause this issue, the mouse pointer works as expected.

Any ideas?

Any help is appreciated.

**Comment by Jorys-Paulin** [comment]
Hey, @MagFlux!
From what I see at first glance it is a problem only occurring on ChromeOS, but does it appends with other applications than Moonlight?

**Comment by MagFlux** [comment]
@Jorys-Paulin, thanks for the quick response!

No, this is the only application I have had this kind of behavior with.

I went ahead and recorded a screen capture at the link below showing the issue:
https://youtu.be/5C1hYXLQjjQ

**Comment by Jorys-Paulin** [comment]
Sorry, @MagFlux, but I honestly don't know what that's about. I'll ask @JacobMix if he has some ideas on that, but I think the problem should append all the time in all fairness, but it doesn't.

**Comment by MagFlux** [comment]
No worries, I appreciate you taking a look at it regardless.

@Jorys-Paulin, to simplify my life is there a specific part of the code base I could take a look at and try to debug locally? Thanks!

**Comment by Jorys-Paulin** [comment]
@MagFlux Not really, it seems your problem comes from your trackpad

**Comment by MagFlux** [comment]
Ah, ok. Well, that's unfortunate.

For whatever it's worth I don't know if you were intentionally being specific or not but this behavior occurs with a USB mouse, Bluetooth mouse, or with the built in trackpad.

**Comment by MagFlux** [comment]
@cgutman I see issues #335 and #353 which are duplicates of this original issue were just closed suggesting the problem was fixed in Chrome Beta (64.0.3282.39) which just became stable. That being said I just went from the special hotfix version you released a while back (which worked) to the latest release of Moonlight at 8.2.0 and the problem still exists for me in Chrome OS (Version 64.0.3282.101 (Official Build) beta (64-bit)). In fact checking back at the Chrome bug relating to this issue (https://bugs.chromium.org/p/chromium/issues/detail?id=781182) which is marked as fixed in M64, I still see the problem in the examples linked to reproduce it so I have no idea whats going on unless I'm misunderstanding how Chrome OS and the browser's versioning works. Now at this point we know its a Chrome issue and not Moonlight's problem so I can understand you wanting to close this issue but I wanted to post this just in case anyone else stumbles upon this problem for the time being.
